### PR TITLE
topaz: Initial support

### PR DESCRIPTION
**Pull request by NikitaSavc**
Initial support for Redmi Note 12 4G (tapas/topaz)

### CODE REVIEW

File: topaz.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"maintainer": "NikitaSavc",
+			"oem": "Redmi",
+			"device": "Note 12 4G",
+                        "filename": "crDroidAndroid-14.0-20240308-topaz-v10.2.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/topaz/10.x/crDroidAndroid-14.0-20240308-topaz-v10.2.zip/download",
+                        "timestamp": 1709901924,
+                        "md5": "fbd688bc7d46c45bb0f5241e480a42ea",
+                        "sha256": "52c7bab8b35fb77f5749adf0a4b249619663e8929b81d403e2ec99c3bfea7bcd",
+                        "size": 1478757188,
+			"version": "v10.2",
```
Comment (gwolf2u) [inline topaz.json]: remove v

File: topaz.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "https://nikgapps.com/crdroid-official",
+			"firmware": "https://xiaomifirmwareupdater.com/archive/firmware/topaz/",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/topaz/10.x/recovery/",
+			"paypal": "",
+			"telegram": "https://t.me/Nikita_Prjkts_Support",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_topaz",
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_topaz"
```
Comment (gwolf2u) [inline topaz.json]: bad url

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: topaz.json (hunk 1 above)
Comment (NikitaSavc) [reply topaz.json]: Resolved

File: topaz.json (hunk 2 above)
Comment (NikitaSavc) [reply topaz.json]: Resolved.

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**